feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go

```diff
@@ -198,23 +201,73 @@ func TestInterfaceLoopbackMode(t *testing.T) {
 	})
 
 	t.Run("Verify AE interface and port-1 are down on DUT", func(t *testing.T) {
-		gnmi.Await(t, dut, gnmi.OC().Interface(aggID).OperStatus().State(), 2*time.Minute, oc.Interface_OperStatus_LOWER_LAYER_DOWN)
-		operStatus := gnmi.Get(t, dut, gnmi.OC().Interface(aggID).OperStatus().State())
-		if want := oc.Interface_OperStatus_LOWER_LAYER_DOWN; operStatus != want {
-			t.Errorf("Get(DUT AE interface oper status): got %v, want %v", operStatus, want)
+
+		want := []oc.E_Interface_OperStatus{oc.Interface_OperStatus_LOWER_LAYER_DOWN, oc.Interface_OperStatus_DOWN}
+		opStatus, statusCheckResult := gnmi.Watch(t, dut, gnmi.OC().Interface(aggID).OperStatus().State(), 2*time.Minute, func(y *ygnmi.Value[oc.E_Interface_OperStatus]) bool {
+			opStatus, ok := y.Val()
+			if !ok {
+				return false
+			}
+			for _, expectedStatus := range want {
+				if opStatus == expectedStatus {
+					return true
+				}
+			}
+			return false
+		}).Await(t)
+		if !statusCheckResult {
+			val, _ := opStatus.Val()
+			t.Errorf("Get(DUT AE interface oper status): got %v, want %v", val.String(), want)
 		}
 
 		gnmi.Await(t, dut, gnmi.OC().Interface(dutPort1.Name()).OperStatus().State(), 1*time.Minute, oc.Interface_OperStatus_DOWN)
-		operStatus = gnmi.Get(t, dut, gnmi.OC().Interface(dutPort1.Name()).OperStatus().State())
+		operStatus := gnmi.Get(t, dut, gnmi.OC().Interface(dutPort1.Name()).OperStatus().State())
 		if want := oc.Interface_OperStatus_DOWN; operStatus != want {
 			t.Errorf("Get(DUT port1 oper status): got %v, want %v", operStatus, want)
 		}
 	})
 
 	t.Run("Configure interface loopback mode FACILITY on DUT AE interface", func(t *testing.T) {
-		gnmi.Update(t, dut, gnmi.OC().Interface(aggID).LoopbackMode().Config(), oc.Interfaces_LoopbackModeType_FACILITY)
-		if deviations.AggregateLoopbackModeRequiresMemberPortLoopbackMode(dut) {
-			gnmi.Update(t, dut, gnmi.OC().Interface(dutPort1.Name()).LoopbackMode().Config(), oc.Interfaces_LoopbackModeType_FACILITY)
+		if deviations.InterfaceLoopbackModeRawGnmi(dut) {
+			gpbSetRequest := &gpb.SetRequest{
+				Update: []*gpb.Update{{
+					Path: &gpb.Path{
+						Origin: "openconfig",
+						Elem: []*gpb.PathElem{
+							{
+								Name: "interfaces",
+							},
+							{
+								Name: "interface",
+								Key: map[string]string{
+									"name": dut.Port(t, "port1").Name(),
+								},
+							},
+							{
+								Name: "config",
+							},
+							{
+								Name: "loopback-mode",
+							},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: []byte("true"),
+						},
+					},
+				}},
+			}
+			gnmiClient := dut.RawAPIs().GNMI().Default(t)
```

Comment:
This line of code failed:

third_party/openconfig/featureprofiles/feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:261:18: not enough arguments in call to dut.RawAPIs().GNMI
	have ()
	want ("testing".TB)
third_party/openconfig/featureprofiles/feature/experimental/interface/interface_loopback_aggregate/otg_tests/interface_loopback_aggregate/interface_loopback_aggregate_test.go:261:39: dut.RawAPIs().GNMI().Default undefined (type "google3/third_party/openconfig/gnmi/proto/gnmi/gnmi_go_proto".GNMIClient has no field or method Default)